.github/workflows/build.yml

```
@@ -2,9 +2,9 @@ name: Build Docker Container & Push to DO Registry
 
 on:
   push:
-    branches: [ main, deployment-digitalocean]
+    branches: [ main, deployment-digitalocean ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)